ARCs/arc-0033.md

```diff
@@ -0,0 +1,60 @@
+---
+arc: 33
+title: Becoming an xGov
+description: Explanation on how to become Expert Governors.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
+xGov status gives a governor the right to vote on proposals raised by the community, to ratify the proposals as valid for final community vote, where said proposals pertain to spending a previously specified amount of Algo in a given epoch on particular initiatives.
+ 
+The purpose of this ARC is to define the standard by which xGov status is achieved.
+ 
+## Specification
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in <a href="https://www.ietf.org/rfc/rfc2119.txt">RFC-2119</a>.
+ 
+### What is an xGov
+Expert Governors (xGov) are decentralized decision makers who demonstrate enduring commitment, are knowledgeable of the realities of the Algorand community, and whose interests are aligned with the good of blockchain.
+ 
+xGov **SHOULD** understand the community's needs and align their interests with those of the Algorand ecosystem.
+xGov **MAY** propose measures for a vote.
+xGov **MAY** participate in the designation of proposals.
+ 
+### Requirement to become an xGov
```

Comment:
This is an interesting point

Reply:
Totally agree. Security of the network is fundamental. Consensus participation shows commitment.

All ALGO committed to governance by xGov should have a mandatory requirement for consensus participation.

Reply:
I also agree, xGov making major decisions should be participating in consensus. This also provides incentive for participating in consensus itself as Ghost has also mentioned (elsewhere)

Reply:
I am against node participation being a requirement. The primary reason is that I think there are a lot of people that deserve a fair chance at being an xGov and don't have the resources or technical knowledge to run a node. Before you say "Oracle Free Tier", I think putting any significant portion of live stake behind Oracle's free tier is a bad idea for a couple of reasons. 

1. They might just stop the free tier at some point. 
2. If node requirements get raised even a bit all of sudden you have a bunch of stake that can no longer participate. 
3. Oracle could ban nodes at any time (see Hetzner). 
4. If the goal is decentralization, putting a portion of live stake online with a company that has contiously been a bad actor in the FOSS space is a bad idea. 

Also, if only node runners are xGovs they will be incentivized to push through proposals that only benefit node runners, but not necessarily the network as a whole.

Reply:
Agree with @GhostOfMcAfee. Participation in consensus is critical for the ecosystem and this would be a reasonable expectation of someone who is trying to guide decision making more.

To @joe-p's points on hardware requirements that is a reasonable concern given that the spirit of Algorand is to make participation as low barrier as possible. However, I think the concerns listed for relying on Oracle/IaaS providers is there for consensus in general and not just in the scope of making it an xGov requirement.

I know the Inc has been very conscientious of making changes that would require higher node requirements, and would make requirements like that known ahead of time for consensus participants to react. 

In my mind xGovs are a bit of representative democracy and overall having them need to demonstrate commitment to the network and lead by example would be beneficial to the ecosystem.

Reply:
To be clear, I'm not against incentivizing nodes. I think it's a great way to bootstrap networks. I just don't think it should be tied to governance. 

Even with hardware requirements being low, there are still plenty of places in the world where they are hard to meet. In the US there are a lot of places that don't have access to quality internet service. Then look at a country like Nigeria, which is strongly adopting Algorand. The new limitations put in place by the central bank alone likely makes it harder to acquire and maintain the resources to run a node, let alone any other factors in the Nigerian economy. The same is true for many developing markets that *need* blockchain technology and I think it's important to give them equitable access to participate in governance.

Reply:
I agree that participating in consensus with a node and a high uptime should be one requirement to become xGov. It shows in a still limited way : technical understanding, technical means, and commitment to the protocol.

Reply:
Agree with @joe-p . Making _running a node_ as a strong requirement would:
* favor technical members
* require a non-technical person to hire some devop guy / or use external service, to run his node

Since there is no minimum amount of Algo required to be a consensus member (or maybe 1 Algo?), I don't think this requirement makes much sense overall. 
On the other hand, it could be a good recommendation to run a consensus node.